autoload/airline/extensions.vim

```diff
@@ -21,8 +21,14 @@ endfunction
 
 let s:script_path = tolower(resolve(expand('<sfile>:p:h')))
 
+if exists('g:NERDTreeStatusline')
```

Comment:
Did you mean to use a different variable name here?
Since `NERDTreeStatusline` is already defined in NERDTree source code, I never get to the else branch of this conditional block, despite I'm not using any custom variable.
`NERDTreeStatusline` defaults to `%{exists('b:NERDTree')?b:NERDTree.root.path.str():''}`

Reply:
If ‘g:NERDTreeSlatusLine’ didn’t defined in my .vimrc, I get to the else block without any problem. 

I will double check this behavior.